content/pages/index.yaml

```diff
@@ -45,16 +50,16 @@ resources:
         - "Community: Social gatherings and hack nights"
         - "Grow: A chance to learn, to mentor, to present"
         - "Network: Find a new team, teammate, or co-founder"
-      - title: Local software meetup group
+      - title: Local Software Meetup Group
         description:
         - "Wider outreach & visibility"
         - "Access to meeting spaces"
         - "Logistical help"
-      - title: Software company
+      - title: Software Company
         description:
         - "Recruiting & hiring"
         - "A way for your employees to grow and learn"
         - "Engagement with the local tech community"
       - title: Entrepreneur
         description:
-        - "Sponsor a meetup (usually between $300-$500) for a chance to pitch to our developers"
+        - "Sponsor a meetup (usually between $300-$500) for a chance to pitch to our developers"
```

Comment:
I think we can maybe drop this. Let's chat in slack

Reply:
This one can be a different PR. That line only changed because it's the new EOF.